bokeh/util/compiler.py

```diff
@@ -527,7 +528,7 @@ def resolve_modules(to_resolve: Set[str], root: str) -> Dict[str, str]:
         for module in to_resolve:
             if module.startswith(("./", "../")):
                 def mkpath(module: str, ext: str = "") -> str:
-                    return abspath(join(root, *module.split("/")) + ext)
+                    return abspath(expanduser(join(root, *module.split("/")) + ext))
```

Comment:
I'm not sure a change here is warranted but @mattpap will have to comment when he returns

Reply:
It's unnecessary, perhaps even unsafe to do so.